string room_name
---
bool success
string msg